client/blocks/signup-form/social.jsx

```diff
@@ -106,9 +105,7 @@ class SocialSignupForm extends Component {
 				} ) }
 			>
 				{ ! compact && (
-					<p className="auth-form__social-text">
-						{ preventWidows( translate( 'Or create an account using:' ) ) }
-					</p>
```

Comment:
Not sure this is still in use as I couldn't find a signup form that displayed it, but should be safe to remove as it's inside a `p`.